File: DataStructures/BinaryTree/simpletree.cs
using System;

public class Node
{
    public int key;
    public Node left, right;

    public Node(int item)
    {
        key = item;
        left = right = null;
    }
}

public class BinaryTree 
{
    Node root;

    BinaryTree(int key)
    {
        root = new Node(key);
    }
    BinaryTree()
    {
        root = null;
    }

    public static void Main(String[] args)
    {
        BinaryTree tree = new BinaryTree(1);
        tree.left = new Node(2);
        tree.right = new Node(3);
        tree.left.right = new Node(4);
    }
}

File: DataStructures/BinaryTree/simpletree.java
public class SimpleTree {

    public class Node 
    {
        int key;
        Node left, right;
        public Node(int k)
        {
            key = k;
            left = right = null;
        }
    }

   public class BinaryTree 
   {

    Node root;
    BinaryTree(int k)
    {
        root = new Node(key);
    }
    BonaryTree()
    {
        root = null;
    }
   }

   public static void main(String[] args)
   {
       BinaryTree tree = new BinaryTree();

       tree.root = new Node(1);

       tree.root.left = new Node(2);
       tree.root.right = new Node(3);

       tree.root.left.left = new Node(4);
   }

}
